### ISSUE TITLE
Bug - Table - Applying info prop causes Th misalignment

### ISSUE THREAD

**Issue by jenny-s51** [patternfly/patternfly-react#11058]
**Describe the problem**
A clear and concise description of the problem. Which components are affected?
See table headers in [table sortable custom control example](https://staging-v6.patternfly.org/components/table#sortable---custom-control).

**How do you reproduce the problem?**
Provide steps to reproduce. A [codesandbox](https://codesandbox.io/s/serverless-cherry-q9t3f ) demonstrating the problem is appreciated.
Remove the `info` prop from the header that applies it. The headers then become properly aligned. 

**Expected behavior**
A clear and concise description of the expected behavior.
Headers should be horizontally aligned. 

**Is this issue blocking you?**
List the workaround if there is one.
Not a blocker. Identified via the MUI theming effort for Kubeflow. Would it be possible to prioritize this for an upcoming sprint @nicolethoen ?

**Screenshots**
If applicable, add screenshots to help explain the issue.

In Kubeflow: 
<img width="805" alt="Screenshot 2024-09-25 at 11 35 44 AM" src="https://github.com/user-attachments/assets/38e30506-52cf-40c4-9ccb-11f15c9f2442">

**What is your environment?**
Chrome, Kubeflow Model Registry

**What is your product and what release date are you targeting?**
Kubeflow Model Registry

**Comment by mcoker** [comment]
@mattnolting will this bug be addressed by https://github.com/patternfly/patternfly/pull/7050?

**Comment by mattnolting** [comment]
@mcoker Yes, a quick copy/paste from the react example on the https://github.com/patternfly/patternfly/pull/7050

![Screenshot 2024-09-25 at 1 23 02 PM](https://github.com/user-attachments/assets/1f36d606-d0f8-4867-a0be-6087cbc0bfae)
